.github/actions/run-e2e-playwright/action.yml

```diff
@@ -29,10 +29,3 @@ runs:
         sleep 90
         npx nx run ${{ inputs.options }}-e2e:e2e
 
-    - name: Upload E2Es results
```

Comment:
Is it intentional?

Reply:
Yes we are taking our report to report portal so it will be duplicate reporting to GHA. for now we only keep this for protractor.